### ISSUE TITLE
[ENHANCEMENT] - Fishing

### ISSUE THREAD

**Issue by troopakoopa** [iNCEPTION-RP/Feedback#11]
**What feature you want to improve**
Fishing

**How could this feature be improved? Please describe your idea as much as possible**

I believe that having trash as a catchable item is terrible, no one wants to go relax with friends fishing an catch trash, out of 50 bait I catch more trash than fish an then you jsut need to dump it on the ground an it stays there till blackout. 

I suggest either remove trash completely or creating a vendor at the trash yard to be able to sell trash to for half of the lowest costing fish. 

**Additional information**

If trash was put in to break up the time between catching fish, I know time was already increased an trash was also added. Between the time of the mini game an also the ability to mess up i think that is enough time between fish i don't think trash is needed at all.

**Comment by TheAlphaStrain** [comment]
I agree. Last time I went fishing, I used 50 bait and ended up with 22 trash. I'm not entirely sure as well why Trash is needed since the weight of the fight alone is enough for you to either stop fishing, store it, then go back to it or go to a fish monger and sell it, ergo also creating time as to not be a job or to grind it out solely for money. Also, within that 50 Bait, I only caught 1 Rainbow Trout? Is it. The most expensive fish. (Not saying I want more of those catches, just that the odds are you won't catch many or you'll catch a few.
